ampd/src/starknet/events/contract_call.rs

```diff
@@ -217,6 +217,27 @@ mod tests {
     }
 
     fn get_dummy_event() -> starknet_core::types::Event {
+        // "hello" as payload
+        // "hello" as destination address
+        // "some_from_address" as source address
+        // "destination_chain" as destination_chain
+        let event_data: Result<Vec<FieldElement>, FromStrError> = vec![
```

Comment:
optional: looks like we could simplify this use case even more by adding a method somewhere that does this:

```
fn prep_field_elements(data: Vec<&str>) -> Vec<FieldElement> {
    data.into_iter().map(FieldElement::from_str).collect().unwrap()
}
```

Reply:
I agree. I'll skip that change, because it's in a lot of places and I want to get this closed and focus your attention on the other PR.
If you insist, I can change it later 🙏